android/app/buildConfigs.gradle

```
@@ -48,7 +53,12 @@ def setupGoogleServices(buildConfig, resValue) {
 }
 
 def setupResourceValues(buildConfigName, resValue) {
-    def command = "yarn --silent babel-node tools/build-config to-xcconfig $buildConfigName"
+    def command
+    if (System.properties['os.name'].toLowerCase().contains('windows')) {
+        command = ["cmd", "/c", "yarn --silent babel-node tools/build-config to-xcconfig $buildConfigName"]
```

```suggestion
        yarnCommand = "yarn --silent babel-node tools/build-config to-xcconfig $buildConfigName"
        command = ["cmd", "/c", yarnCommand]
```

Also the yarn command could be deduplicated.